python/test/pipeline_test.py

```diff
@@ -28,6 +28,7 @@
 import unittest
 import urllib
 import urlparse
+import pytz
```

Comment:
This is a third-party library, right? it is already bundled with the AE SDK/runtime?